src/api/app/models/notification.rb

```diff
@@ -19,6 +24,27 @@ class Notification < ApplicationRecord
   scope :for_web, -> { where(web: true) }
   scope :for_rss, -> { where(rss: true) }
 
+  scope :read, -> { where(delivered: true) }
+  scope :unread, -> { where(delivered: false) }
+
+  scope :comments, -> { where(notifiable_type: 'Comment') }
+  scope :requests, -> { where(notifiable_type: 'BsRequest') }
+  scope :with_notifiable, -> { where.not(notifiable_id: nil).where.not(notifiable_type: nil) }
+  scope :without_notifiable, -> { where(notifiable_id: nil, notifiable_type: nil) }
```

Comment:
Those really should not exist anymore right?

Reply:
Removed